packages/@react-stately/table/src/useTableState.ts

```diff
@@ -64,19 +64,21 @@ export function useTableState<T extends object>(props: TableStateProps<T>): Tabl
   );
   let {disabledKeys, selectionManager} = useGridState({...props, collection});
 
-  return {
+  let state = useMemo(() => ({
```

Comment:
Memoized a couple of state hooks so that rerenders of the TableView wouldn't cause rerenders of the cells (since the state won't have changed and thus useContext won't cause a rerender)